archaius2-archaius1-bridge/README.md

```diff
@@ -22,9 +22,58 @@ Injector injector = Guice.createInjector(
 
 ## How does it work
 
-The bridge calls ConfigurationManager.install() to configure the legacy API with an 
-AbstractConfiguration implementation that bridges the static API with the Guice created 
-Archaius2 top level Config binding.  Note that this implementation can only work if 
-ConfigurationManager is allowed to bootstrap itself with the default AbstractConfiguration 
-instance.  The bridge is instantiated via static injection which occurs very early in 
-the Guice bootstrapping process, specifically before most singletons are created.  
+The bridge calls ConfigurationManager.install() to configure the legacy API with an AbstractConfiguration implementation that bridges the static API with the Guice created Archaius2 top level Config binding.  Note that this implementation can only work if ConfigurationManager is allowed to bootstrap itself with the default AbstractConfiguration instance.  The bridge is instantiated via static injection which occurs very early in the Guice bootstrapping process, specifically before most singletons are created.  
+
+## Troubleshooting
+
+The static bridge can be a bit fragile as it tries to 'fix' the static usage pattern of the old API with dependency injection.  As such static access of ConfigurationManager before Guice has finished bootstrapping will result in an IllegalStateException.  
+
+### IllegalStateException("Archaius2 bridge not usable because ConfigurationManager was initialized too early.  See stack trace below.")
```

Comment:
The wording no longer matches the header. It might be better to use the `...` here like you did on `Not using expected bridge!!!`.